app/plugin_api.go

```diff
@@ -426,6 +426,41 @@ func (api *PluginAPI) GetEmojiImage(emojiId string) ([]byte, string, *model.AppE
 	return api.app.GetEmojiImage(emojiId)
 }
 
+// Plugin Section
+
+func (api *PluginAPI) GetPlugins() ([]*model.Manifest, *model.AppError) {
+	plugins, err := api.app.GetPlugins()
```

Comment:
Can this logic stay there or should it be moved to `app/plugins.go`?

Reply:
It can stay here but I'd suggest adding a small test since it's more than just 1-2 lines

Reply:
Is there an easier way to set up plugins in a test environment then
```go
th := Setup().InitBasic()
defer th.TearDown()
setupPluginApiTest(t, "InsertGoCodeHere", "InsertManifestHere", "id", th.App)
``` 
? Is there some kind of sample Plugin which is reused in multiple tests?

Reply:
@jwilander Friendly ping ^

Reply:
No, I don't think there are any tests that do that at the moment. If you can see a better way to test these, then feel free to propose something.

Reply:
Sorry @hanzei for the late response, basically what @crspeller said